test/0.4.24/contracts/SecondOpinionOracle__Mock.sol

```diff
@@ -0,0 +1,30 @@
+// SPDX-License-Identifier: UNLICENSED
+// for testing purposes only
+
+pragma solidity 0.8.9;
+
+contract SecondOpinionOracle__Mock {
```

Comment:
Not sure that we need that mock. We can harness OracleReportSanityChecker and just return true on _askSecondOpinion.

Reply:
?

Reply:
Done in e9a302d4175e9a3217d30594bdd58d834c7452c8